rnote-ui/src/appwindow/appwindowactions.rs

```diff
@@ -537,6 +540,27 @@ impl RnAppWindow {
             }),
         );
 
+        // Remove page from doc in fixed size mode
+        action_remove_page_from_doc.connect_activate(
+            clone!(@weak self as appwindow => move |_action_remove_page_from_doc, _target| {
+                let canvas = appwindow.active_tab().canvas();
+
+                let format_height = canvas.engine().borrow().document.format.height;
+                let format_width = canvas.engine().borrow().document.format.width;
+                let doc_height = canvas.engine().borrow().document.height;
+                let new_doc_height = doc_height - format_height;
+                if doc_height > format_height {
+                    let aabb = Aabb::new(na::point![0.0, new_doc_height], na::point![0.0 + format_width, doc_height]);
+                    canvas.engine().borrow_mut().store.split_colliding_strokes(aabb, aabb);
+                    let remove_area = canvas.engine().borrow_mut().store.stroke_keys_as_rendered_in_bounds(aabb);
```

Comment:
let's name this a bit differently to avoid confusion, I'd append `_keys` so: `remove_area_keys`

Reply:
Done